.circleci/docker/Dockerfile.ubuntu2004

```diff
@@ -21,67 +21,33 @@
 #
 # (c) 2016-2019 solidity contributors.
 #------------------------------------------------------------------------------
-FROM buildpack-deps:disco AS base
+FROM buildpack-deps:focal AS base
 
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN set -ex; \
-	dist=$(grep DISTRIB_CODENAME /etc/lsb-release | cut -d= -f2); \
-	echo "deb http://ppa.launchpad.net/ethereum/cpp-build-deps/ubuntu $dist main" >> /etc/apt/sources.list ; \
-	apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1c52189c923f6ca9 ; \
 	apt-get update; \
 	apt-get install -qqy --no-install-recommends \
 		build-essential \
 		software-properties-common \
-		cmake ninja-build clang++-8 libc++-8-dev libc++abi-8-dev \
+		cmake ninja-build \
 		libboost-filesystem-dev libboost-test-dev libboost-system-dev \
 		libboost-program-options-dev \
-		libjsoncpp-dev \
-		llvm-8-dev libcvc4-dev libz3-static-dev libleveldb1d \
+		libcvc4-dev z3 libz3-dev \
 		; \
-	apt-get install -qy python-pip python-sphinx; \
-	update-alternatives --install /usr/bin/llvm-symbolizer llvm-symbolizer /usr/bin/llvm-symbolizer-8 1; \
-	pip install codecov; \
+	apt-get install -qy python3-pip python3-sphinx; \
+	pip3 install codecov; \
```

Comment:
We do still have nightly codecov runs that probably need this, don't we? I was considering to remove it...

Reply:
I don't need them.

Reply:
Well, I haven't looked at them either, since they weren't really helpful last time I checked ages ago... should I just remove them and the nightly code coverage runs then, while I'm at it?

Reply:
Ping @axic for the opportunity to object ;-).

Reply:
I object, but didn't wanted to be suspicious :wink:

I think code coverage serves a different purpose than the idea to "assign a number to each error message and check each of them is triggered". That idea won't show any uncovered code paths, or will it?

Reply:
> Well, I haven't looked at them either, since they weren't really helpful last time

The only problem was that syntax and semantic coverage was mixed into one. If that is still a problem, we could just use it for syntax, to have clarity over coverage of syntax tests. Once sufficiently high, go over to semantic coverage.

Reply:
Ok, given that I'd say we keep this in here in the docker image for now and decouple the rest of the coverage discussion from this particular PR. [marking as resolved ;-)]